README.md

```diff
@@ -264,7 +264,7 @@ Flags:
 
 ### status
 
-View the status of a tileset. This includes how many jobs are queued, processing, and complete.
+View the status of a tileset. To get more detailed information about a tileset's jobs, including the timestamps of failed and successful jobs, use the `tielsets jobs <tileset_id>` command.
```

Comment:
typo tielsets -> tilesets

Maybe we can make the tileset-jobs relationship even more explicit with something like
```suggestion
View the status of the most recent job for a tileset. To get more detailed information about a tileset's jobs, including the timestamps of failed and successful jobs, use the `tilesets jobs <tileset_id>` command.
```